packages/fx-core/src/plugins/resource/bot/constants.ts

```diff
@@ -125,6 +125,7 @@ export class ErrorNames {
   public static readonly MSG_ENDPOINT_UPDATING_ERROR = "MessageEndpointUpdatingError";
   public static readonly COMMAND_EXECUTION_ERROR = "CommandExecutionError";
   public static readonly CALL_APPSTUDIO_API_ERROR = "CallAppStudioAPIError";
+  public static readonly API_ERROR = "ApiError";
```

Comment:
It seems that the telemetry is tracking app studio api error, can it reuse the above CALL_APPSTUDIO_API_ERROR ?

Reply:
Removed the `ApiError` error event.